MANIFEST.in

```diff
@@ -1,9 +1,13 @@
-exclude .gitignore
+exclude .cookiecutter-replay.json
 exclude .darglint
 exclude .editorconfig
-exclude noxfile.py
+exclude .gitignore
 exclude CODEOWNERS
+exclude CONTRIBUTING.md
+exclude mkdocs.yml
+exclude noxfile.py
 exclude src/conftest.py
 recursive-exclude .github *
+recursive-exclude docs *
 recursive-exclude tests *
-recursive-exclude benchmarks *
```

Comment:
We no longer exclude the benchmarks?

Reply:
Yes, from the source distribution package.